tests/helpers/test_config_validation.py

```diff
@@ -1953,3 +1953,30 @@ async def test_is_entity_service_schema(
         vol.All(vol.Schema(cv.make_entity_service_schema({"some": str}))),
     ):
         assert cv.is_entity_service_schema(schema) is True
+
+
+def test_renamed(caplog: pytest.LogCaptureFixture, schema) -> None:
```

Comment:
Where is `schema` defined?

Reply:
it's a fixture, it's defined a bit earlier in the module

Reply:
Ok, I looked at the top of the module where we usually put module level fixtures, and didn't find it.